src/runtime/tests/manifest-test.ts

```diff
@@ -3764,15 +3765,76 @@ recipe Three
     assert.equal(multiParam.params['foo'], 'Text');
     assert.equal(multiParam.params['bar'], 'Number');
     assert.equal(multiParam.params['baz'], 'Boolean');
-    assert.deepEqual(multiParam.targets, ['Store', 'Handle', 'HandleConnection', 'Schema', 'SchemaField']);
+    assert.deepEqual(multiParam.targets, ['Store', 'Handle', 'HandleConnection', 'Schema', 'SchemaField', 'Recipe']);
     assert.equal(multiParam.retention, 'Runtime');
 
     const goodForAll = manifest.annotations['goodForAll'];
     assert.isEmpty(goodForAll.params);
     assert.isEmpty(goodForAll.targets);
     assert.equal(goodForAll.retention, 'Runtime');
 
-    // TODO: store and serialize manifest items' annotations
-    // assert.equal(manifest.toString(), manifestStr);
+    assert.equal(manifest.toString(), manifestStr);
+
+    for (const recipe of manifest.recipes) {
+      const cloneRecipe = recipe.clone();
+      assert.equal(cloneRecipe.toString(), recipe.toString());
+    }
+  });
+  it('verifies referencing annotations', async () => {
```

Comment:
I don't understand what this means... 

Also, it might be nicer to split up all these assertions below into their own test cases. e.g. "throws when annotation not defined", "thrown when target incorrect", etc.

Reply:
done